client_test/crash_test.go

```diff
@@ -84,7 +84,7 @@ func TestCrashTolerance(t *testing.T) {
 }
 
 func directlyDefundALedgerChannel(t *testing.T, alpha client.Client, beta client.Client, channelId types.Destination) {
-	id := alpha.CloseLedgerChannel(channelId)
+	id, _ := alpha.CloseLedgerChannel(channelId)
```

Comment:
We could check the returned error and fail the test on error. This comment also applies to other changes in test files.

Reply:
Good call. Fixed each of these tests to inspect the returned error